indico/modules/rb/client/js/common/filters/FilterDropdown.jsx

```diff
@@ -151,6 +151,7 @@ export default class FilterDropdown extends React.Component {
     return (
       <Popup
         position="bottom left"
+        className="ui form"
```

Comment:
This seems to break some of the styling :/

![image](https://github.com/user-attachments/assets/10ed1938-af01-4fdb-b371-282db1364764)

Reply:
Ugh...